test/fixtures/inner-locals.html

```diff
@@ -0,0 +1,7 @@
+<h1>index</h1>
+
+<include src="./test/fixtures/includes/3.html">
+  {
+    "text": 3
+  }
+</include>
```

Comment:
This test will fail with an error `Unexpected token s in JSON at position 3`
```html
<include src="./test/fixtures/includes/3.html">
  simple: fail
</include>
```

https://github.com/posthtml/posthtml-include/blob/99549766beebc1acbc567da0e7fdc4209e17d6db/lib/index.js#L28-L29

I think it's worth changing the validation logic, something like this

```js
if (locals) {
  try {
    locals = JSON.parse(locals);
  } catch {
    locals = false;
  }
}
```

You also forgot to add this to the main `test/test.js` file
```js
test('Inner locals', t => {
  return process(t, 'inner-locals')
})
```

Reply:
Thats true, i fixed it. I also made a tiny change on the logic when trying to parse "locals" to fit this needings. All the tests are passing now.

Reply:
add this test more coverage
```html
<include src="./test/fixtures/includes/3.html">
  simple: fail
</include>
```